_rules/form-field-visual-label-is-descriptive-1cd4da.md

```diff
@@ -0,0 +1,210 @@
+---
+id: 1cd4da
+name: Form field visual label is descriptive
+rule_type: atomic
+description: |
+  This rule checks that visual labels describe the purpose of form field elements.
+accessibility_requirements:
+  wcag21:2.4.6: # Headings and labels (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Anne Thyme Nørregaard
+  - Dagfinn Rømen
```

Comment:
Following the discussion in the ACT-Rules meeting Thursday 12.09.2019, please consider taking Geir Sindre and my name off this rule. The rule has been under major revision since we last authored it.